### ISSUE TITLE
No release file

### ISSUE THREAD

**Issue by qaqak** [anbox/anbox#2048]
1. Please check that no similar bug is already reported. Have a look on the list of open bugs at https://github.com/anbox/anbox/issues
2. Make sure you are running the latest version of Anbox before reporting an issue. Update snap to latest: `snap refresh --devmode --edge anbox`
3. Make sure you have debug logs enabled:
`sudo snap set anbox debug.enable=true`
4. Reproduce the error while debug logs enabled.
5. Run the anbox logs collection utility and attach the tar file.
`sudo /snap/bin/anbox.collect-bug-info `

6. ** Please paste the result of `anbox system-info` below:**
<details>
<summary>anbox system-info output</summary>

```
Command 'anbox' not found, but can be installed with:
sudo apt install anbox

```
</details>

**Please describe your problem:**
The PPA for the Anbox kernel modules has no Release file. APT won't let me install the modules because of that

**What were you expecting?:**
The PPA to have a Release file

**Additional info:**
Using a clean install of Pop!_OS Jammy Jellyfish

**Comment by Fuseteam** [comment]
the PPA is no longer needed since 19.04

**Comment by qaqak** [comment]
But they should keep the PPA available if some Ubuntu-based distros don't come with the binder and ashmem modules

**Comment by Fuseteam** [comment]
no, ubuntu-based distros likely use the ubuntu-kernel, debian has had those modules build-in for years now
by the time debian drops the modules, that's the release it will needed for

**Comment by Fuseteam** [comment]
you don't, instead you poke the people over at @pop-os at https://github.com/pop-os/linux/issues/113
